### ISSUE TITLE
Rust

### ISSUE THREAD

**Issue by burdges** [dedis/purb#14]
Is anyone doing a Rust implementation?

**Comment by nikirill** [comment]
No, not at the moment. But if there is concrete interest, we could probably look into that.
